DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/ManageCourseController.cs

```diff
@@ -303,10 +304,28 @@ private int GetIntegerFromStringOrConvertToZeroIfNull(string? input)
             EditCourseDetailsFormData formData
         )
         {
+            if (customisationName == string.Empty)
+            {
+                if (courseService.DoesCourseNameExistAtCentre(
+                    customisationId,
+                    "",
+                    centreId,
+                    formData.ApplicationId
+                ))
+                {
+                    ModelState.AddModelError(
+                        nameof(EditCourseDetailsViewModel.CustomisationName),
+                        "A course with this name already exists, add on to the course name to make it unique"

```

Comment:
The reason for this error message is because if the user deletes the customisation name and there is already an course with the same application name and no customisation name, it is then required to specify a customisation name. It looked a bit weird to me to just say "a course with this name already exists" when the box the error is highlighting has nothing in it. Since the course name we're talking about is the uneditable application name. Can you think of a better way to phrase it?
![image](https://user-images.githubusercontent.com/70278044/144033880-af4f6ef0-9b96-4a91-86e5-9ee18b1ce3ec.png)